modules/transaction-payment/src/lib.rs

```diff
@@ -305,6 +314,18 @@ pub mod module {
 
 		/// Weight information for the extrinsics in this module.
 		type WeightInfo: WeightInfo;
+
+		/// Treasury PalletId used to derivate sub account.
+		#[pallet::constant]
+		type TreasuryPalletId: Get<PalletId>;
```

Comment:
```suggestion
		type PalletId: Get<PalletId>;
```

This is pallet ID of this pallet, not the treasury pallet. We want to avoid any possible confusion.